doc/api/addons.markdown

```diff
@@ -767,7 +793,7 @@ void MyObject::New(const FunctionCallbackInfo<Value>& args) {
     double value = args[0]->IsUndefined() ? 0 : args[0]->NumberValue();
     MyObject* obj = new MyObject(value);
     obj->Wrap(args.This());
-    args.GetReturnValue().Set(args.This());
+    return;
```

Comment:
> Fine for me anyhow. First I was wondering though why args.GetReturnValue().Set(args.This()); is not necessary anymore, but I may have missed API changes.
Same here, but https://github.com/nodejs/node-addon-examples/pull/49#discussion_r52022800 says that with a construct call the return value is always `this`.



> Please remove it, I think it is confusing.

Sure :]